src/main/scala/cromwell/engine/package.scala

```diff
@@ -30,6 +30,8 @@ import scala.util.{Failure, Success, Try}
  * Internally, this package is built on top of [[cromwell.binding]].
  */
 package object engine {
+  private val DEFAULT_CALL_CACHING_VALUE = false
```

Comment:
oh whoops I meant `DefaultCallCachingValue`, all caps with underscores is tres Java.  :smile: